db/migrate/20240201014607_marketplace_add_servings_to_products.rb

```diff
@@ -0,0 +1,5 @@
+class MarketplaceAddServingsToProducts < ActiveRecord::Migration[7.1]
+  def change
+    add_column :marketplace_products, :servings, :integer, null: true
```

Comment:
Hmmm, serving size is very often a range. What about supporting that in next iteration? Could have some kind of composite property of `servings_min` and `servings_max`?

Reply:
I think there's a couple things at play:

- Human-Servings (i.e. "It feeds 2 to 4, depending on appetite")
- FDA-Servings (i.e. "250g per serving, provides this many calories")

I think I'd rather focus my energies on widening the useful information (like the tags for Vegan, GF, etc) unless we get feedback from the client otherwise.